src/components/Contacts/ContactDetails/ContactDetailsHeader/ContactHeaderSection/ContactHeaderAddressSection.tsx

```diff
@@ -51,17 +55,24 @@ export const ContactHeaderAddressSection = ({
 
       return (
         <ContactHeaderSection icon={<LocationIcon />}>
-          <Typography variant="subtitle1">{envelope}</Typography>
-          <Typography variant="subtitle1">{street}</Typography>
+          <StyledAddressTypography variant="subtitle1">
+            {envelope}
+          </StyledAddressTypography>
+          <StyledAddressTypography variant="subtitle1">
+            {street}
+          </StyledAddressTypography>
           <Box>
-            <Typography
+            <StyledAddressTypography
               style={{ display: 'inline-block' }}
               variant="subtitle1"
-            >{`${city}, ${state ?? ''} ${postalCode}`}</Typography>{' '}
+            >{`${city}${city && ','} ${
```

Comment:
We checked that `city` is truthy on line 52, so `${city && ','}` can just be `,` right?